<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tools_meminspector</class>
 <widget class="QDialog" name="tools_meminspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory Inspector</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>568</width>
     <height>212</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutMemInspect">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <layout class="QFormLayout" name="formLayoutIndicies">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_BeginIndex"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_EndingIndexLE">
        <property name="text">
         <string>Ending Index:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_EndingIndex"/>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayoutUtil">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Read Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_1B">
          <property name="text">
           <string>1 byte</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2B">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>2 bytes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_4B">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>4 bytes</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Display">
          <property name="text">
           <string>Display</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_BeginIndexLE">
        <property name="text">
         <string>Beginning Index:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeWidget" name="treeWidget_MemInspect">
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lineEdit_BeginIndex</zorder>
  <zorder>verticalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
